CuPy installation found, continuing using GPU acceleration.
having the CPUs {0, 2, 4} available.
Started basinhopping at 2021-01-16 21:21:54.267851 UTC, with
command line arguments = {'path': '/home/jkgpu/HVQE/data/chain/periodic/own_graph/ops/energy/18', 'n_par': 26, 'par_multiplicity': 9, 'n_iter': 0, 'cost_fn': 'energy', 'temperature': 1.0, 'stepsize': 1.0, 'init_par': None, 'dump_interval': None}
runtime variables =  {'GPU': True, 'date_start': '2021-01-16 21:21:54.267851', 'complete_graph': [(1, 2), (3, 4), (5, 6), (7, 8), (9, 10), (11, 12), (13, 14), (15, 16), (17, 0), (0, 1), (2, 3), (4, 5), (6, 7), (8, 9), (10, 11), (12, 13), (14, 15), (16, 17)], 'init_layer': [(0, 1), (2, 3), (4, 5), (6, 7), (8, 9), (10, 11), (12, 13), (14, 15), (16, 17)], 'layers': [[(1, 2), (3, 4), (5, 6), (7, 8), (9, 10), (11, 12), (13, 14), (15, 16), (17, 0)], [(0, 1), (2, 3), (4, 5), (6, 7), (8, 9), (10, 11), (12, 13), (14, 15), (16, 17)]], 'n': 18}
.........................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................../home/jkgpu/miniconda3/lib/python3.8/site-packages/chainer/backends/cuda.py:142: UserWarning: cuDNN is not enabled.
Please reinstall CuPy after you install cudnn
(see https://docs-cupy.chainer.org/en/stable/install.html#install-cudnn).
  warnings.warn(
/home/jkgpu/miniconda3/lib/python3.8/site-packages/chainer/_environment_check.py:70: UserWarning: 
--------------------------------------------------------------------------------
CuPy (cupy) version 8.0.0 may not be compatible with this version of Chainer.
Please consider installing the supported version by running:
  $ pip install 'cupy>=6.3.0,<7.0.0'

See the following page for more details:
  https://docs-cupy.chainer.org/en/latest/install.html
--------------------------------------------------------------------------------

  warnings.warn(msg.format(
 
==========================================================================================
Finished basinhopping of  /home/jkgpu/HVQE/data/chain/periodic/own_graph/ops/energy/18 at 2021-01-16 21:58:41.190621 UTC, with
 
{'path': '/home/jkgpu/HVQE/data/chain/periodic/own_graph/ops/energy/18', 'n_par': 26, 'par_multiplicity': 9, 'n_iter': 0, 'cost_fn': 'energy', 'temperature': 1.0, 'stepsize': 1.0, 'init_par': None, 'dump_interval': None}
init_par = [0.0004042139775788773, -0.0004182117876425342, 0.0001544630336190138, 0.0003614819772521467, -0.00013815401887640189, 0.0004275688123659327, 0.00021142496069361612, -0.00018232528834791342, -0.00025130276608866355, 0.00038091551990577567, -0.0003335501143674079, -0.00032293942861855, -9.75602646918327e-07, 0.00017843833613522897, -3.834159354671509e-05, 9.568165375585405e-05, -2.1367552737432648e-05, -0.0004432301285229505, 0.000363689957972916, 0.00028470636683338314, -1.3652246124245995e-05, 0.00011726361940631025, -0.00044183395260871195, -0.00031844624430362443, 0.0001290505982280137, 0.00038452263746928937]
 
RESULTS:
--------
E_VQE = -8.015020159573867
inf_VQE = 0.0068958392423695836
n_fn_calls = 555
Wall-clock time 0.6128007335795297 hours
 
For more output, see /home/jkgpu/HVQE/data/chain/periodic/own_graph/ops/energy/18/output.txt
Plots of output saved in /home/jkgpu/HVQE/data/chain/periodic/own_graph/ops/energy/18
End of HVQE.py
==========================================================================================
